src/App.vue

```diff
@@ -2,10 +2,12 @@
   <main>
     <Navbar></Navbar>
     <RouterView />
+    <Footer></Footer>
   </main>
 </template>
 
 <script setup>
 import Navbar from "@/components/Navbar.vue";
 import {RouterView} from 'vue-router'
+import Footer from "./components/Footer.vue";
```

Comment:
Let's try to stick with @ annotation for the rest of the project. Very minor but better to follow project structure.